src/index.html

```diff
@@ -5,8 +5,360 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html" class="header__logo">
+        <img
+          src="images/logo.png"
+          alt="Moyo_Logo"
+          class="header__logo-img"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list-item ">
+            <a href="index.html" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__list-item">
+            <a href="index.html" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__list-item">
+            <a href="index.html" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+              href="index.html"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a href="index.html" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__list-item">
+            <a href="index.html" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__list-item">
+            <a href="index.html" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__list-item">
+            <a href="index.html" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+
+      <div class="card" data-qa="card">
+        <img
+          class="card__photo"
+          src="images/imac.jpeg"
+          alt="product-photo"
+        >
+
+        <h2 class="card__descr">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="card__rating-descr">Reviews: 5</div>
```

Comment:
Need to use the semantic tag for text, fix it everywhere
```suggestion
          <p class="card__rating-descr">Reviews: 5</p>
```